thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2722,6 +2723,19 @@ void CMasterGraph::handleSlaveDone(unsigned node, MemoryBuffer &mb)
         sdMb.setBuffer(len, (void *)d);
         act->slaveDone(node, sdMb);
     }
+    Owned<IThorActivityIterator> iter = getConnectedIterator();
+    ForEach(*iter)
+    {
+            CGraphElementBase &element = iter->query();
+            CActivityBase *activity = element.queryActivity();
+            if (activity)
+            {
+                stat_type nodeSpillSize = activity->getNodeSpillSize(node);
```

Comment:
minor: see prev. comment, as this is only ever in master context, getNodeSpillSize should prob. be a non-virtual in CMasterActivity and 'actvitiy' cast here to access it.